node/hubController.js

```diff
@@ -326,7 +327,7 @@ class HubController {
     }
 
     _createTranslator(translatorName, deviceInfo) {
-        this.ConsoleLogger.info(`_createTranslator ${translatorName}`);
+        this.logger.info(`_createTranslator ${translatorName}`);
         return this.OpenT2T.createTranslatorAsync(translatorName, deviceInfo).then( translator => {
```

Comment:
If I understand this correctly, it sounds like each of the public entry points should take a logger as a parameter, is that correct?

---
In reply to: [109985895](https://github.com/openT2T/cloud/pull/32#discussion_r109985895) [](ancestors = 109985895,109985579)